src/NServiceBus.AmazonSQS.AcceptanceTests/NServiceBus.AmazonSQS.AcceptanceTests.csproj

```diff
@@ -8,15 +8,23 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)Test.snk</AssemblyOriginatorKeyFile>    
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Workaround for https://github.com/nunit/nunit3-vs-adapter/issues/296 -->
+    <DebugType Condition="'$(TargetFramework)' != '' AND '$(TargetFramework)' != 'netcoreapp11'">Full</DebugType>
```

Comment:
> since this is a test lib i think the condition is redundant. just always make it full.

👍 

the extra property group was meant for an easy removal instead of having the change nested with other settings.